### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jessicaschilling**
Updating Companion repo readme as part of general cleanup under https://github.com/ipfs-shipyard/ipfs-companion/issues/896. Final piece of that to-do list, so ...

Closes https://github.com/ipfs-shipyard/ipfs-companion/issues/896

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@lidel, are you able to take a quick look? Nothing major here, just a lot of text tightening. Also updated all GIFs to current Companion version under Mac. Thanks!

File: README.md [hunk 1]
```diff
@@ -11,160 +8,142 @@
...
-- [Privacy Policy](#privacy-policy)
-- [License](#license)
+- [Help & Troubleshooting](#help--troubleshooting)
+- [Privacy & Licenses](#privacy--license)
 
-## Background
+## About IPFS Companion
 
-This add-on enables everyone to access IPFS resources the way they were meant: from locally running IPFS node :-)
+IPFS Companion harnesses the power of your locally running IPFS node (either through the IPFS Desktop app or the command-line daemon) directly inside your favorite Chromium-based or Firefox browser, enabling support for `ipfs://` addresses, automatic IPFS gateway loading of websites and file paths, easy IPFS file import and sharing, and more.
```
Comment (lidel) [inline README.md:34]: Some users may learn that Desktop exists from this very paragraph. 
We should add link – would github one be enough?

```suggestion
IPFS Companion harnesses the power of your locally running IPFS node (either through the [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop/#ipfs-desktop) app or the command-line daemon) directly inside your favorite Chromium-based or Firefox browser, enabling support for `ipfs://` addresses, automatic IPFS gateway loading of websites and file paths, easy IPFS file import and sharing, and more.
```

File: README.md [hunk 2]
```diff
@@ -11,160 +8,142 @@
...
 
-IPFS is a new hypermedia distribution protocol, addressed by content and identities.
-IPFS enables the creation of completely distributed applications.
-It aims to make the web faster, safer, and more open.
+IPFS is a peer-to-peer hypermedia protocol designed to make the web faster, safer, more resilient, and more open. It enables the creation and dissemination of completely distributed sites and applications that don’t rely on centralized hosting and stay true to the original vision of an open, flat web. Visit [the IPFS Project website](https://ipfs.io) to learn more.
 
-Learn more at [ipfs.io](https://ipfs.io) (it is really cool, we promise!)
+## IPFS Companion features
```
Comment (lidel) [inline README.md:38]: Link in ToC at the top is broken due to this change – unsure which one should be changed, up to you

File: README.md [hunk 3]
```diff
@@ -11,160 +8,142 @@
...
-### Automagical Detection of IPFS Resources
+#### Detect URLs with IPFS paths
 
-#### IPFS Path in URL
-
-Requests for IPFS-like paths (`/ipfs/{cid}` or `/ipns/{peerid_or_host-with-dnslink}`) are detected on any website.  
-If tested path is a [valid IPFS address](https://github.com/ipfs/is-ipfs) it gets redirected and loaded from a local gateway, e.g:  
+IPFS Companion detects and tests requests for IPFS-like paths ( such as `/ipfs/{cid}` or `/ipns/{peerid_or_host-with-dnslink}`) on any website. If a path is a [valid IPFS address](https://github.com/ipfs/is-ipfs), it is redirected to load from your local gateway:  
 > `https://ipfs.io/ipfs/QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR`  
 > → `http://127.0.0.1:8080/ipfs/QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR`
```
Comment (lidel) [inline README.md:46]: We switched the default from IP to hostname to enable subdomain support, examples should reflect that:
```suggestion
> → `http://localhost:8080/ipfs/QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR`
> → `http://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi.ipfs.localhost:8080`
```

File: README.md [hunk 4]
```diff
@@ -11,160 +8,142 @@
...
 
-#### DNSLink
+#### Detect DNSLink-enabled URLs
 
-Companion will detect presence of [DNSLink](https://docs.ipfs.io/guides/concepts/dnslink/) in DNS records of visited websites and redirect HTTP request to a local gateway.
+IPFS Companion [detects DNSLink info](http://docs.ipfs.io/how-to/dnslink-companion/) in the DNS records of websites. If a site uses DNSLink (a few examples are https://docs.ipfs.io, https://ipld.io, and http://tr.wikipedia-on-ipfs.org), IPFS Companion redirects the HTTP request to your local gateway:
 
 > `http://docs.ipfs.io`  
 > → `http://127.0.0.1:8080/ipns/docs.ipfs.io`
```
Comment (lidel) [inline README.md:53]:
```suggestion
> → `http://localhost:8080/ipns/docs.ipfs.io` → `http://docs.ipfs.io.ipns.localhost:8080/`
```

File: README.md [hunk 5]
```diff
@@ -11,160 +8,142 @@
...
+### Automatically detect and redirect IPFS resources
 
-### Automagical Detection of IPFS Resources
+#### Detect URLs with IPFS paths
 
-#### IPFS Path in URL
-
-Requests for IPFS-like paths (`/ipfs/{cid}` or `/ipns/{peerid_or_host-with-dnslink}`) are detected on any website.  
-If tested path is a [valid IPFS address](https://github.com/ipfs/is-ipfs) it gets redirected and loaded from a local gateway, e.g:  
+IPFS Companion detects and tests requests for IPFS-like paths ( such as `/ipfs/{cid}` or `/ipns/{peerid_or_host-with-dnslink}`) on any website. If a path is a [valid IPFS address](https://github.com/ipfs/is-ipfs), it is redirected to load from your local gateway:
```
Comment (lidel) [inline README.md:44]: We should  mention Origin isolation on `localhost` (feel free to reword)
```suggestion
IPFS Companion detects and tests requests for IPFS-like paths ( such as `/ipfs/{cid}` or `/ipns/{peerid_or_host-with-dnslink}`) on any website. If a path is a [valid IPFS address](https://github.com/ipfs/is-ipfs), it is redirected to load from your local gateway. Gateway  at `localhost` will automatically switch to subdomain to provide unique Origin per website:
```

File: README.md [hunk 6]
```diff
@@ -11,160 +8,142 @@
...
+
+IPFS Companion enables you to quickly and easily access common actions from your browser bar with just a few clicks:
+
+- See how many peers you're connected with a glance at the cube icon in your browser bar
+- Check your IPFS API and gateway status by clicking the cube icon to reveal the main menu
+- Right-click images and other page assets to easily add them to IPFS (including the option to preserve file names)
+- Choose the _Share files via IPFS_ option in the main menu for quick drag-and-drop import from a browser tab
+- Pin or unpin IPFS resources (via API) directly from the main menu
+- Copy shareable public gateway links, IPFS content paths, or CIDs of IPFS resources directly from the main menu
+- Launch the [IPFS Web UI dashboard](https://github.com/ipfs/webui/) from the main menu with a single click
```
Comment (lidel) [inline README.md:86]:
```suggestion
- Launch the [IPFS Web UI dashboard](https://github.com/ipfs-shipyard/ipfs-webui) from the main menu with a single click
```

File: README.md [hunk 7]
```diff
@@ -11,160 +8,142 @@
...
     - `ipfs://$cid`
     - `ipns://$cid_or_fqdn`
     - `dweb:/ipfs/$cid`
     - `dweb:/ipns/$cid_or_fqdn`
-- Make plaintext IPFS links clickable ([demo](https://ipfs.io/ipfs/bafybeidvtwx54qr44kidymvhfzefzxhgkieigwth6oswk75zhlzjdmunoy/linkify-demo.html))
-- Switch between _External_ HTTP API and _Embedded_ js-ipfs node. Read about differences at [docs/node-types](docs/node-types.md).
-  > [![screenshot of node type switch](https://user-images.githubusercontent.com/157609/42382479-b4d98768-8134-11e8-979c-69b758846bf0.png)](https://github.com/ipfs-shipyard/ipfs-companion/blob/master/docs/node-types.md)
 
-## Install
+- Switch between the external HTTP API of your local IPFS node (default setting) and a js-ipfs node embedded in your browser (note that this has some [functionality limitations](docs/node-types.md))
```
Comment (lidel) [inline README.md:118]:
```suggestion
- Switch between the external HTTP API of your local IPFS node (default setting) and a js-ipfs node embedded in your browser (note that this has some [functionality limitations](https://docs.ipfs.io/how-to/companion-node-types/))
```

File: README.md [hunk 8]
```diff
@@ -11,160 +8,142 @@
...
-**Note:** `ipfs-companion` is designed to retrieve content from a locally running IPFS daemon.  
-Make sure [IPFS is installed](https://docs.ipfs.io/introduction/usage/) on your computer.
-
-### Beta Channel
-
-Developers and enthusiasts can opt-in for Beta-quality channel with hand-picked Dev Builds:
-
-- Beta for Firefox: [Self-hosted Signed Dev Build](https://ipfs.io/ipfs/QmR8W5wg8BuAyBTruHnHovfWRavwvidVh3qtyinXi6NnLa)
-- Beta for Chromium-based browsers: [Dev Build at Chrome Web Store](https://chrome.google.com/webstore/detail/ipfs-companion-dev-build/hjoieblefckbooibpepigmacodalfndh)
+- [Firefox self-hosted signed dev build](https://ipfs.io/ipfs/QmR8W5wg8BuAyBTruHnHovfWRavwvidVh3qtyinXi6NnLa)
```
Comment (lidel) [inline README.md:135]: While we are at it, let's switch to subdomain link, that way Firefox user grants extension install permission only to this one CID, not entire gateway :grimacing: 
```suggestion
- [Firefox self-hosted signed dev build](https://bafybeibjozlsoxzrxsoklis775aglnwpal2hjl42ippo57jdwiv6zoij7m.ipfs.dweb.link/)
```

File: README.md [hunk 9]
```diff
@@ -11,160 +8,142 @@
...
+- [Chromium-based dev build at the Chrome Web Store](https://chrome.google.com/webstore/detail/ipfs-companion-dev-build/hjoieblefckbooibpepigmacodalfndh)
 
-It is also possible to [grab the last successful build from `master`](https://ci.ipfs.team/job/IPFS%20Shipyard/job/ipfs-companion/job/master/lastSuccessfulBuild/),
-but these builds are not signed nor will automatically update:
-`.zip` bundles are meant only to be manually loaded via `chrome://extensions` (Chromium-based) or `about:debugging` (Firefox) for the purpose of quick smoke-testing.
+It's also possible to grab the [last successful build from `master`](https://ci.ipfs.team/job/IPFS%20Shipyard/job/ipfs-companion/job/master/lastSuccessfulBuild/),
```
Comment (lidel) [inline README.md:138]: Hm.. we don't use `ci.ipfs.team`, but we could point at releases instead?

```suggestion
It's also possible to grab [vendor-specific packages for each release](https://github.com/ipfs-shipyard/ipfs-companion/releases),
```

File: README.md (hunk 1 above)
Comment (jessicaschilling) [reply README.md:34]: It's the best we've got right now 😊 sounds good.

File: README.md (hunk 3 above)
Comment (jessicaschilling) [reply README.md:46]: Oh drat -- thanks for catching that.

File: README.md (hunk 9 above)
Comment (jessicaschilling) [reply README.md:139]: Sounds good.

File: README.md (hunk 7 above)
Comment (jessicaschilling) [reply README.md:119]: Oh boo, thought I'd found all those. Thanks.

**Comment by jessicaschilling** [comment]
All resolved - thank you! Merging.

**Pull request merged**